PlaygroundService/appsettings.json

```diff
@@ -2,7 +2,8 @@
   "ContractSetting": {
     "ContractClassPath": "/Users/jasonlu/playground-service/HelloWorldContract/src/",
     "StateClassPath": "/Users/jasonlu/playground-service/HelloWorldContract/src/",
-    "ProtoPath": "/Users/jasonlu/playground-service/HelloWorldContract/src/Protobuf/contract/"
+    "ProtoPath": "/Users/jasonlu/playground-service/HelloWorldContract/src/Protobuf/contract/",
+    "MaxFileSizeMB": 50

```

Comment:
This should not be under contract settings. Can you create another subgroup? Thanks.